sdk/schemaregistry/azure-data-schemaregistry/README.md

```diff
@@ -12,7 +12,7 @@ schema identifiers rather than full schemas.
 
 - A [Java Development Kit (JDK)][jdk_link], version 8 or later.
 - [Azure Subscription][azure_subscription]
-- An [Event Hubs namespace][event_hubs_namespace]
+- An [Event Hubs schema registry][event_hubs_namespace]
```

Comment:
Should we refer to this as `Event Hubs Schema Registry` or just `Schema Registry`? Also, fix the link.
```suggestion
- An [Event Hubs schema registry][event_hubs_schema_registry]
```